lib/SIL/SILFunctionBuilder.cpp

```diff
@@ -76,31 +76,11 @@ SILFunctionBuilder::addFunctionAttributes(SILFunction *F, SILDeclRef constant,
       // Either only adjoint is specified, or both primal and adjoint are
```

Comment:
Remove mentions of `primal` and `adjoint`, remove `primName` and `adjName` below.
Also, `spcified` -> `specified`.